sympy/crypto/crypto.py

```diff
@@ -298,6 +333,9 @@ def encipher_vigenere(pt, key, symbols="ABCDEFGHIJKLMNOPQRSTUVWXYZ"):
 
             ``m``: string of upper-case letters (the plaintext message)
 
+            ``repeats``: True if repetition of characters in key is allowed.
+                         Otherwise false.
```

Comment:
This doesn't really explain what happens for false. It sounds like it would give an error.